GraphicsToolsUnityProject/ProjectSettings/GraphicsSettings.asset

```diff
@@ -3,7 +3,7 @@
 --- !u!30 &1
 GraphicsSettings:
   m_ObjectHideFlags: 0
-  serializedVersion: 14
+  serializedVersion: 13
```

Comment:
Did you mean to change this file?

Reply:
You can revert these files via:
`git checkout upstream/main GraphicsToolsUnityProject/ProjectSettings`